whedon.rb

```diff
@@ -13,11 +13,14 @@
 
 set :views, Proc.new { File.join(root, "responses") }
 
-config_file 'config/settings.yml'
+config_file "config/settings-#{ENV['RACK_ENV']}.yml"
 set :configs, {}
 
 # 'settings.journals' comes from sinatra/config_file
+# FIXME: Don't initialize settings unless production
+
 settings.journals.each do |journal|
+  next unless ENV['RACK_ENV'] == "production"
```

Comment:
Have to skip this because when we `require` the app in `spec_helper.rb` this code is called which means the GitHub API interactions below (`params["editors"] = github_client.team_members(team_id).collect { |e| e.login }.sort`) get executed.